bin/configure-tower-projects.py

```diff
@@ -215,7 +215,7 @@ def extract_emails(self, arns: Sequence[str]) -> List[str]:
                 email = match.group("session_name")
                 emails.append(email)
             else:
-                raise ValueError(
+                print(
```

Comment:
Will the new ARN format cause issues in this tower config script?

Reply:
The new ARN format will cause this print statement to output, but they will otherwise be ignored. That's why they'll have to be added to the Tower workspace manually, as per the checkbox in my PR description. This is intended to be a short-term fix until we can consolidate user information into a single config file as per WORKFLOWS-453.

Reply:
Ah yes - should have read the description. 🙃 Thanks Bruno!